swagger.yaml

```diff
@@ -2094,6 +2094,53 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /liveiso:
```

Comment:
This is a live ISO, but we already have another live ISO that runs the installer agent.  How about we call this one the installer ISO to avoid confusion?

Reply:
Done.